typings/index.d.ts

```diff
@@ -14,11 +14,9 @@ declare module 'klasa-dashboard-hooks' {
 		public server: Server;
 		public routes: RouteStore;
 		public middlewares: MiddlewareStore;
-		public dashboardUsers: DataStore<string, DashboardUser, typeof DashboardUser>;
+		public dashboardUsers: BaseManager<string, DashboardUser, typeof DashboardUser>;
 	}
 
-	export { DashboardClient as Client };
-
```

Comment:
Why did you remove these lines?

Reply:
I receive this error in both Visual Studio Code and Webstorm.

![image](https://user-images.githubusercontent.com/53883918/85586182-b2930b80-b60e-11ea-83cd-bb16c3c843e7.png)